### ISSUE TITLE
[wavedeform] example is broken (Trac #1195)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1189]
It tries to run WaveCalibrator, but the syntax is wrong.  Fixing that produces other errors, so I think something changed within the last few years.

Migrated from https://code.icecube.wisc.edu/ticket/1195
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:57", 
    "description": "It tries to run WaveCalibrator, but the syntax is wrong.  Fixing that produces other errors, so I think something changed within the last few years.", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067117911749", 
    "component": "combo reconstruction", 
    "summary": "[wavedeform] example is broken", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2015-08-19T17:11:14", 
    "milestone": "", 
    "owner": "jbraun", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150820T15:01:17: 
* jbraun commented:

> In changeset 136621:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="136621"
> Remove obsolete WaveCalibrator configuration.  Closes #1189.
> ```


* jbraun changed resolution from "" to "fixed"
* jbraun changed status from "new" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:57: 
* olivas changed milestone from "IceRec V5" to ""
* olivas commented:

> Milestone IceRec V5 deleted
